roles/mongodb/meta/main.yml

```diff
@@ -16,3 +16,6 @@ galaxy_info:
         - 7
   categories:
     - system
+dependencies:
+  - role: epel
```

Comment:
Should have conditional check: Include `epel` only when `ansible_os_family == "RedHat"`.